### PR TITLE
CTCP-1897: Update response to reflect spec

### PR DESCRIPTION
**Pull request by drnaylor**

### CODE REVIEW

File: app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala [hunk 1]
```diff
@@ -33,6 +33,6 @@ class MessagesController @Inject() (cc: ControllerComponents)(implicit val mater
   def post(): Action[Source[ByteString, _]] = Action(streamFromMemory) {
     request: Request[Source[ByteString, _]] =>
       request.body.runWith(Sink.ignore)
```
Comment (liandra-giacomine) [inline app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala:48]: Do we not want some type of verification here for expected headers etc to ensure that the router is sending the right things through?

Comment (sonisn1988) [reply app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala:48]: I think better to verify all the headers send from router to make sure everything passed from router is received at stub or EIS end.

Comment (drnaylor) [reply app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala:48]: Yeah, I don't mind doing that, this was just the minimum viable change.

**Review by sonisn1988 — APPROVED** [review]
LGTM. I think we can handle one 500 error scenario with some different ticket. As once we send data to EIS there are possibilities we get 400 or 500 from EIS. But as per spec we need to convert any 400 or 500 with 500 so better to have one error scenario for 500 as well.

**Comment by drnaylor** [comment]
>  think we can handle one 500 error scenario with some different ticket. As once we send data to EIS there are possibilities we get 400 or 500 from EIS. But as per spec we need to convert any 400 or 500 with 500 so better to have one error scenario for 500 as well.

A 500 error is a "we did something wrong, it wasn't your fault", so it should _never_ return in response to anything we do. Therefore, returning a 500 is pointless.

Also, this is a stub, not a test harness, and we are using this in live scenarios. It's meant to simulate the real world. Nothing we send should cause a 500 error.

If we want to test the effect of a 500, that's for the integration tests in the router, and we do that there.

**Pull request merged**